internal/jimm/jimm.go

```diff
@@ -135,6 +157,10 @@ func (j *JIMM) GetRoleManager() RoleManager {
 	return j.RoleManager
 }
 
+func (j *JIMM) GetGroupManager() GroupManager {
```

Comment:
Don't start getters with "Get", it is "neither idiomatic nor necessary" according to effective Go:
https://go.dev/doc/effective_go#Getters

Reply:
It can't be named GroupManager because there is a field named GroupManager. Once we get rid of the large jimm interface, we can delete these getters.